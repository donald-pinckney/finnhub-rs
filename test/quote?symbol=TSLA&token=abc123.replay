{"request":{"url":"https://finnhub.io/api/v1/quote?symbol=TSLA&token=abc123","method":"GET","body":null,"headers":{}},"response":{"url":"https://finnhub.io/api/v1/quote?symbol=TSLA&token=abc123","status":200,"headers":{"access-control-allow-credentials":"true","access-control-allow-headers":"Origin","access-control-allow-methods":"POST, GET, OPTIONS, PUT, DELETE","access-control-allow-origin":"*","access-control-expose-headers":"Content-Length","alt-svc":"h3-27=\":443\"; ma=86400, h3-28=\":443\"; ma=86400, h3-29=\":443\"; ma=86400","cf-cache-status":"DYNAMIC","cf-ray":"644c3cf74fc8304f-ORD","cf-request-id":"09a3846e890000304f2784c000000001","connection":"keep-alive","content-type":"application/json; charset=utf-8","date":"Sat, 24 Apr 2021 03:27:50 GMT","expect-ct":"max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"","nel":"{\"max_age\":604800,\"report_to\":\"cf-nel\"}","report-to":"{\"group\":\"cf-nel\",\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report?s=SYilB%2BsYhhp7xO7bRPYdWdua7sJWrLy0zPHjz0%2Fu0qwueIqi%2BcL7zs6m118kbhVMUULmS5XPcvIY22paI7oPfcko8AbAmirMCwZ0qhOzu%2BtplVU%2FGWfS\"}],\"max_age\":604800}","server":"cloudflare","set-cookie":"__cfduid=d0b5883b246c3e5fde1b1b25dc48095d31619234870; expires=Mon, 24-May-21 03:27:50 GMT; path=/; domain=.finnhub.io; HttpOnly; SameSite=Lax; Secure","transfer-encoding":"chunked","x-ratelimit-limit":"60","x-ratelimit-remaining":"51","x-ratelimit-reset":"1619234930"},"body":"eyJjIjo3MjkuNCwiaCI6NzM3LjM1OSwibCI6NzE1LjQ2LCJvIjo3MTkuOCwicGMiOjcxOS42OSwidCI6MTYxOTIwODAwMn0="},"format_version":3}