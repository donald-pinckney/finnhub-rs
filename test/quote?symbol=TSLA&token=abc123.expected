CompanyQuote {
    c: 729.4,
    h: 737.359,
    l: 715.46,
    o: 719.8,
    pc: 719.69,
    t: 1619208002,
}
